sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -110,10 +117,16 @@ def test_windowed_batches(self):
           | util.BatchElements(
               min_batch_size=5, max_batch_size=10, clock=FakeClock())
           | beam.Map(len))
-      assert_that(res, equal_to([
-          5, 5, 10, 10,  # elements in [0, 30)
-          10, 7,         # elements in [30, 47)
-      ]))
+      assert_that(
+          res,
+          equal_to([
+              5,
+              5,
+              10,
+              10,  # elements in [0, 30)
```

Comment:
Nit: I prefer the previous formatting (as the comment applies to the whole set of elements).

Reply:
Reverted.